BossMod/Modules/RealmReborn/Trial/T09WhorleaterH/T09WhorleaterHBodyslam.cs

```diff
@@ -0,0 +1,95 @@
+using BossMod.Components;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace BossMod.Modules.RealmReborn.Trial.T09WhorleaterH;
+    class BodySlamKB : Knockback
+    {
+        private float Distance;
+        private Angle Direction;
+
+        private float LeviathanZ;
+
+        public override IEnumerable<Source> Sources(BossModule module, int slot, Actor actor)
+        {
+            if (Distance > 0)
+                yield return new(new(), Distance, default, null, Direction, Kind.DirForward);
```

Comment:
First argument should probably be `module.Bounds.Center`?

Reply:
I don't know. This part was written by Taurenkey and is only marked as new since I split it into multiple files.
And I am not sure if the attack is originating from center or from opposite edge of the arena. But the code worked in my tests.